OSDOCS-14809: Capacity Reservations in ROSA HCP (#100445)
openshift/openshift-docs: rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc

You can edit machine pool configuration options such as scaling, adding node labels, and adding taints.

ifdef::openshift-rosa-hcp[]
You can also create new machine pools with Capacity Reservations.

.Overview of AWS Capacity Reservations

> If you have reserved compute capacity using link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/capacity-reservation-overview.html[AWS Capacity Reservations] for a specific instance type and Availability Zone, you can use it for your {product-title} worker nodes. Both On-Demand Capacity Reservations and Capacity Blocks for ML are supported.

Author: Left a question above. Please, see the 1st comment on this diff page.

> You purchase a Capacity Reservation and manage its life cycle directly with AWS. After reserving the capacity, pass a Capacity Reservation ID when creating a new machine pool in your {product-title} cluster. You can also use a Capacity Reservation shared with you from another AWS Account within your AWS Organization.

Reviewer: ```suggestion
Purchase and manage a Capacity Reservation directly with AWS. After reserving the capacity, pass a Capacity Reservation ID when creating a new machine pool in your {product-title} cluster. You can also use a Capacity Reservation shared with you from another AWS account within your AWS organization.
```
I'm not sure I understand what "pass a Capacity Reservation ID" means.

> Once you configure Capacity Reservations in {product-title}, you can use AWS to monitor reserved capacity usage across all workloads in your AWS Account.

Author: We could probably say:

Once you configure Capacity Reservations in {product-title}, you can use your AWS account to monitor reserved capacity usage across all workloads in your account.

@bchandra-ocp, can you please confirm this will be an accurate thing to say? Thank you.

Using Capacity Reservations on machine pools in {product-title} clusters has the following prerequisites and limitations:

> * You have installed the {rosa-cli} of version 1.2.57 or above.

Reviewer: ```suggestion
* You have installed the {rosa-cli} version 1.2.57 or above.
```

* Your {product-title} cluster is of version 4.19 or later.
* Your cluster already has a machine pool that is not using a Capacity Reservation or taints. This machine pool must have at least 2 worker nodes.

> * You have purchased a Capacity Reservation for the instance type required in the Availability Zone of the machine pool that you are creating.

Reviewer: ```suggestion
* You have purchased a Capacity Reservation for the instance type required in the AZ of the machine pool that you are creating.
```
